src/ucp/core/ucp_mm.h

```diff
@@ -31,6 +36,7 @@ typedef struct ucp_mem {
     ucs_memory_type_t   mem_type;       /* Type of allocated or registered memory */
     ucp_md_index_t      alloc_md_index; /* Index of MD used to allocated the memory */
     ucp_md_map_t        md_map;         /* Which MDs have valid memory handles */
+    uint32_t            flags;
```

Comment:
move after alloc_md_index
+ pls add ucp_mem to object sizes gtest and output of `ucx_info -y` if not there already